### DISCUSSION TITLE
What is the difference between FragPipe and DIA-NN?

### DISCUSSION THREAD

**Discussion by akila-wijerathna-yapa** [vdemichev/DiaNN#979] [Q&A] [answered]
I am familiar with DIA-NN. However I have been given a Linux HPC and not familiar with CL and unable to run DIA-NN since  None of the Linux version comes with a GUI so its CL only.  However I got to know FragPipe https://fragpipe.nesvilab.org/ in the apps available via Open OnDemand (OOD) which as DIA-NN as part of its pipeline and it comes with a GUI.

Is it possible to only use DIA-NN features from FragPipe? Anyone has a manual for this?

**Comment by vdemichev** [comment, marked as answer]
Hi Akila,

Just a quick comment about Linux.
It does not come with the GUI indeed, but the command line interface is exactly the same as under Windows. Hence you can see how the GUI configures the required commands under Windows and just use the same ones under Linux - just copy the command line the GUI prints and paste it in the Linux terminal or (better) make a bash script that invokes it (this is rather simple). You can even run the Windows GUI under Linux using Wine.

The purpose of DIA-NN in the FragPipe is not to be used as a standalone tool but rather to do the 'second pass' MBR search downstream of MSFragger-DIA, which is a faster tool for large search space analyses.

Best,
Vadim
